docs/resources/Channel.md

```diff
@@ -332,6 +332,34 @@ Discord utilizes a subset of markdown for rendering message content on its clien
 
 Using the markdown for either users, roles or channels will mention the target(s) accordingly.
 
+## URL Formatting
+
+Discord utilizes the id's and hashes from specific objects to provide the images rendered in the client. This functionality uses the following formats:
+
+###### Image Formats
+
+| Name | Extension |
+|-------|------------|
+| JPEG | .jpg, .jpeg |
+| PNG | .png |
+| WebP | .webp |
+| GIF | .gif ([user](#DOCS_USER/user-object) avatars only) |
+
+###### Image Sizes
+
+Power of 2 between 16 and 1024.
```

Comment:
For User Avatar and Guild Icon, the ones who can have these "variables" sizes, 1024 is the maximum that they increase in size from my tests.